azurerm/resource_arm_iothub_test.go

```diff
@@ -298,3 +325,34 @@ resource "azurerm_iothub" "test" {
 }
 `, rInt, location, rStr, rInt)
 }
+
+func testAccAzureRMIotHub_fallbackRoute(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_iothub" "test" {
+  name                  = "acctestIoTHub-%d"
+  resource_group_name   = "${azurerm_resource_group.test.name}"
+  location              = "${azurerm_resource_group.test.location}"
+
+  sku {
+    name     = "S1"
+    tier     = "Standard"
+    capacity = "1"
+  }
+
+  fallback_route {
+    source         = "DeviceMessages"
+    endpoint_names = ["fallback"]
```

Comment:
For the test to pass, this is needed
```suggestion
    endpoint_names = ["events"]
```

Reply:
Well that was it! thanks